sconfig/shared/src/main/scala/org/ekrich/config/impl/ConfigDelayedMergeObject.scala

```diff
@@ -144,22 +144,22 @@ final class ConfigDelayedMergeObject(
     else false
   }
   override def hashCode: Int = stack.hashCode
-  override def render(
+  override def renderAtKey(
```

Comment:
Not sure when formatting options should change Merge versions of config rendering.

Reply:
So you are saying you have a merge so now you have a combined document? Output here is not something to save but just to view?

Reply:
Look at class hierarchy, and where are current formatting changes.
For now I don't know what for are other classes. Like those with Merge in the name of class.